scripts/py_dependency.py

```diff
@@ -0,0 +1,102 @@
+import sys
+from pathlib import Path
+from io import StringIO
+
+def add_to_dict(dictionary, key, val):
+        if key in dictionary:
+            dictionary[key].add(val)
+        else:
+            dictionary[key] = {val}
+
+def print_help():
```

Comment:
Please see the comment about sys.argparse. This whole function will be deleted then.